### PR TITLE
feat: Tabs support indicatorLength

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/44311

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Tabs support `indicatorLength` to customize length of indicator.         |
| 🇨🇳 Chinese |    Tabs 组件新增 `indicatorLength` 属性，用于自定义指示条的长度       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 873f687</samp>

This pull request adds a new prop `indicatorLength` to the `Tabs` component, which allows users to customize the length of the indicator line. It also updates the `rc-tabs` dependency and the documentation in both English and Chinese.

### <samp>🤖 Generated by Copilot at 873f687</samp>

* Add a new prop `indicatorLength` to the `Tabs` component to customize the length of the indicator line ([link](https://github.com/ant-design/ant-design/pull/44406/files?diff=unified&w=0#diff-55d7b2e14d2f70c11368544596a23fa53e1c2d392b85295e52cbf9d21bec7ebcR55), [link](https://github.com/ant-design/ant-design/pull/44406/files?diff=unified&w=0#diff-aae9511c7fe785a991682a0bb026fec2a489547d55493b5fd8264299cd290b15R57))
* Update the dependency version of `rc-tabs` to 12.11.0 to support the new prop ([link](https://github.com/ant-design/ant-design/pull/44406/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L151-R151))
* Format the `package.json` file with `prettier` ([link](https://github.com/ant-design/ant-design/pull/44406/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L335-R335))

### CODE REVIEW

File: components/tabs/index.zh-CN.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
 | centered | 标签居中展示 | boolean | false | 4.4.0 |
 | defaultActiveKey | 初始化选中面板的 key，如果没有设置 activeKey | string | `第一个面板` |  |
 | hideAdd | 是否隐藏加号图标，在 `type="editable-card"` 时有效 | boolean | false |  |
+| indicatorLength | 自定义指示条长度，默认与 tab 等宽 | number \| (origin: number) => number | - | 5.9.0 |
```
Comment (afc163) [inline components/tabs/index.zh-CN.md:57]: 这居然是个属性，为啥不用 token 来搞

Comment (afc163) [reply components/tabs/index.zh-CN.md:57]: 如果需求是比文字短 32px，怎么设置？

Comment (MadCcc) [reply components/tabs/index.zh-CN.md:57]: 因为指示条的长度是自动匹配 tab 宽度的，不是纯样式。

> 如果需求是比文字短 32px，怎么设置？

```
indicatorLength={(origin: number) => origin - 32}
```

Comment (dengfuping) [reply components/tabs/index.zh-CN.md:57]: - 建议 ConfigProvider 支持配置 Tabs 的 `indicatorLength`，因为一般都是全局设置。
- 如果支持 ConfigProvider，还需要同步到 Card tabList

File: components/dropdown/__tests__/__snapshots__/demo-extend.test.ts.snap [hunk 2]
```diff
@@ -5269,6 +5269,7 @@ Array [
       </div>
       <li
         class="ant-dropdown-menu-item-divider"
+        role="separator"
```
Comment (afc163) [inline components/dropdown/__tests__/__snapshots__/demo-extend.test.ts.snap:5272]: 为啥会动这个。

**Review by afc163 — APPROVED** [review]

File: components/dropdown/__tests__/__snapshots__/demo-extend.test.ts.snap (hunk 2 above)
Comment (MadCcc) [reply components/dropdown/__tests__/__snapshots__/demo-extend.test.ts.snap:5272]: 之前 rc-menu 一直没升上来，现在 rc-tabs 带着一起升上来了

**Pull request merged**